### DISCUSSION TITLE
Is There A Way To Read Node Firmware Revision Using Phone App?

### DISCUSSION THREAD

**Discussion by ikearat** [meshtastic/Discussions#59] [Android]
I thumbed around for awhile and the only place I can read the Firmware installed on my node is on the little attached display, briefly while it it rebooting. Is there a way to read the firmware version using the phone UI? Thanks. I know this is going to be about the Android app and you can read the Android app version, but when I tried to use the Android app to read the node firmware I failed.
Reason I ask, I just found out that a release 2.5.7 (Android) was made yesterday (Nov 9, 2024) while reading this thread:
https://github.com/meshtastic/Meshtastic-Android/issues/1372

**Comment by BluegrassMesh** [comment]
Assuming you are using the Android app and connecting to your node via bluetooth. You should be able to see it on the screen where you select the node to connect to after it connects and pulls down the config. 

![4_github](https://github.com/user-attachments/assets/c1c568bf-14b3-4554-97db-f463c112c881)

**Reply by ikearat** [reply]
Yes! Thank you!
I almost never consider that screen once connected and while connecting have paid no attention to the "connected to radio" dialogue, focused instead on the pick-a-dot feature to confirm connection... then it's away from there, only returning when I lose connection and have to delete the paired bluetooth and reestablish it.
Thank you very much!

**Comment by dtgolder** [comment]
Thanks for asking...I've been looking high and low for the firmware version also..

**DEVS:** I might suggest the firmware should have a more prominent home--perhaps the "Device" submenu under the "Radio Configuration" menu?

**Reply by garthvh** [reply]
It is on the details view for the node in on iOS and could be added on Android

**Reply by dtgolder** [reply]
Adding it to Android would be wonderful (gave up on iPhone quite a few years ago).

**Reply by mariotti** [reply]
Hoi, why not a new sub-menu with THIS NODE INFO with: Node ID, MAC added to it. (Version already there)
I add it as a comment as this discussions seems resolved. But in general I had issues to find node specific infos.
For example the TZ settings.
Thanks to all for the opportunity :)

**Reply by garthvh** [reply]
MAC address is not available anymore firmware version is available for the connected node and any that you can administer
